packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,555 @@
+| Breadcrumb        | Wrapper for the Breadcrumb component. Contains `nav` and `ol` elements.  |
+| BreadcrumbDivider | Divider component                                                        |
+| BreadcrumbItem    | `li` element. Can contain BreadcrumbButton or BreadcrumbLink components. |
+| BreadcrumbLink    | Breadcrumb Link                                                          |
+| BreadcrumbButton  | Breadcrumb Button                                                        |
+
+### Breadcrumb
+
+#### Anatomy
+
+![visual anatomy of the Breadcrumb component](./assets/breadcrumb-anatomy.png)
+
+#### DOM
+
+```HTML
+<nav aria-label="breadcrumb">
+  <ol>
+  {children}
+  </ol>
+</nav>
+```
+
+#### Breadcrumb structure:
+
+- root `nav` element
+- list `ol` element
+
+#### API
+
+| Property   | Values                                  | Default       | Purpose                                     |
+| ---------- | --------------------------------------- | ------------- | ------------------------------------------- |
+| appearance | `transparent`, `subtle`                 | `transparent` | Sets appearance                             |
+| focusMode  | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`         | Sets the focus behavior for the Breadcrumb. |
+| size       | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb              |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button - BreadcrumbButton component is used inside BreadcrumbItem.
+- Link - BreadcrumbLink is used inside BreadcrumbItem.
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbButton onClick={() => console.log('smth...')}>
+    Item 2
+  </BreadcrumbButton>
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbLink href="#">
+    Item 3
+  <BreadcrumbLink>
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default  | Purpose                                     |
+| ------------ | --------------------------------------- | -------- | ------------------------------------------- |
+| active       | boolean                                 | false    | Indicates if the item is active             |
+| disabled     | boolean                                 | false    | Disables Breadcrumb item                    |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`    | Sets the focus behavior for the Breadcrumb. |
```

Comment:
The amount of options here is confusing to me what are the diffeerences between

* no-tab vs tab-exist
* off vs tab-only

Reply:
Also why is the `focusMode` set on the BreadcrumbItem? 

```tsx
<Breadcrumbs>
  <BreadcrumbItem focusMode="off" />
  <BreadcrumbItem focusMode="no-tab" />
  <BreadcrumbItem focusMode="tab-exit />
  <BreadcrumbItem focusMode="tab-only />
</Breadcrumbs>
```
The above would be rather confusing 😅

Reply:
I'll ask designers if we need at all additional configuration for the focus. And yeah, it should be in the root component :)

Reply:
Focus will be turned-on by default and no other configuration should be there